src/components/LoginForm/LoginForm.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+import { ViewIcon, ViewOffIcon } from '@chakra-ui/icons';
+import {
+  Button,
+  FormControl,
+  FormHelperText,
+  FormLabel,
+  Input,
+  InputGroup,
+  InputRightElement,
+  Stack
+} from '@chakra-ui/react';
+export const LoginForm = () => {
+  const [show, setShow] = React.useState(false);
+  const handleClick = () => setShow(!show);
+
+
+  const {
+    register,
+    handleSubmit,
+    watch,
+    reset,
+    formState: { errors }
+  } = useForm({
+    reValidateMode: 'onSubmit',
+    mode: 'onTouched'
+  });
+
+  const email = watch('email');
+  const password = watch('password');
+  console.log(watch());
+  const isValid = email && password;
```

Comment:
better to have an isEmpty flag
```suggestion
  const isEmpty = !email || !password;
```